emcc.py

```diff
@@ -2029,6 +2029,9 @@ def compile_source_file(i, input_file):
 
     ## Continue on to create JavaScript
 
+    if shared.Settings.STANDALONE_WASM:
+      linker_inputs.insert(0, system_libs.Library.get_usable_variations()['crt1'].get_path())
```

Comment:
this might be simpler in system_libs, where we already have
```
  if shared.Settings.STANDALONE_WASM:
    add_library(system_libs_map['libstandalonewasm'])
```

Reply:
I think this is the only outstanding issue? lgtm with this moved to system_libs.

Reply:
The reason I did it this was is that crt1 normally wants to come before any other object files on the link line, so it would need to be prepended to the link command line whereas `system_libs.calculate` results are added to the end.

However, remembered the lld works more like the windows linker where the order isn't so important and it can find symbols in anywhere on the command line (it doesn't do "left to right" linking like the GNU linker).